server/odc-service/src/main/java/com/oceanbase/odc/metadb/connection/DatabaseSpecs.java

```diff
@@ -74,4 +76,9 @@ public static Specification<DatabaseEntity> projectIdIsNull() {
         return (root, query, builder) -> builder.isNull(root.get("projectId"));
     }
 
+    public static Specification<DatabaseEntity> typeIn(List<DatabaseType> types) {
+        return (root, query, builder) -> CollectionUtils.isEmpty(types) ? builder.conjunction()
```

Comment:
A little tips: Using `SpecificationUtil.columnIn()` and field name defined in `XxxEntity_`.

Reply:
Nice suggestion~ I updated it.